packages/cli/src/commands/test.ts

```diff
@@ -181,7 +181,13 @@ export default class Test extends AuthCommand {
         return filterByCheckNamePattern(grep, check.name)
       })
       .filter(([, check]) => {
-        return filterByTags(targetTags?.map((tags: string) => tags.split(',')) ?? [], check.tags)
+        const tags = check.tags ??= []
```

Comment:
Why did you go with `??=`? This assigns an empty array to check.tags if it is empty first and then assigns the same value to the `tags` variable. I'm curious if there was a special reason

Reply:
Then I would suggest just using `??` to not modify the check and groups data, even though it wouldn't be a big problem 🤷‍♂️

Reply:
yup, did that, thanks for the tip!